gcp/kubernetes/README.md

```diff
@@ -121,3 +179,22 @@ export JIANT_PATH="/nfs/jiant/home/$USER/jiant"
 ```
 
 You should see your job as `<job_name>` in Kubernetes Engine -> Workloads, and can monitor status, resource usage, and logs from that page.
+
+There are also additional options, such as for sending jobs to different GPU types, and deleting jobs (pods in Kubernetes terminology).
+
+======
+
+### Using an existing GKE cluster
+
+If someone has already set up a GKE cluster for jiant, and you have a different set of things to do:
+
+1. Create a lightweight GCP instance and SSH into it. git clone jiant.
```

Comment:
=> install jiant. (In case anyone's working from a release package or pip.)

Reply:
I don't believe jiant needs to be installed at this point. Really this cloning step is just to get the `mount_nfs.sh` script and related config files.
(Or are you referring to using an installation of jiant as opposed to a git cloned copy?)

Reply:
Yeah. Maybe change this to something like 'clone or download a copy of jiant'?